api/proto/teleport/legacy/client/proto/authservice.proto

```diff
@@ -2296,8 +2296,113 @@ message ExportUpgradeWindowsResponse {
   string SystemdUnitSchedule = 3;
 }
 
+// AssistantMessageRequest is a request to the assistant service.
+message AssistantMessageRequest {
+  // conversationId it's the ID of a conversation.
```

Comment:
```suggestion
  // ConversationId identifies a conversation.
```

So that the generated godocs match the name of the generated type.

Reply:
Fixed